exercises/info.se

```diff
@@ -0,0 +1,9 @@
+;; -*- dune -*-
+
+(exercise
```

Comment:
I don't understand how sexplib works with dune, does dune also compile this when I run dune build?

Reply:
Sexplib is "just" a library of parsers / converters for the s-expression format. So this file currently get loaded at run-time by some logic in `exercise.ml`, and the converters are mostly auto-derived with a PPX. I was mostly copying Rustlings' [`info.toml`](https://github.com/rust-lang/rustlings/blob/main/info.toml) idea :-)

(The `;; -*- dune -*-` bit at the top is just to borrow syntax highlighting options from Dune files.)